payjoin/README.md

```diff
@@ -0,0 +1,87 @@
+<div align="center">
+  <h1>Payjoin</h1>
+
+  <img src="../static/monad.svg" width="150" />
```

Comment:
relative path causes a break in the crates.io docs

see https://crates.io/crates/payjoin

prefer

```suggestion
  <img src="https://raw.githubusercontent.com/payjoin/rust-payjoin/refs/heads/master/static/monad.svg" width="150" />
```